### ISSUE TITLE
WCAG 2.3.1

### ISSUE THREAD

**Issue by MrGrigri** [amark/gun#1079]
The main site, could cause photosensitive epilepsy. 

I noticed that each quadrant, when moused over, causes the background to change. If a user wants to learn about the documentation and the mouse starts in the upper right. when they move to the lower left to click the button, the background flashes more than three times in under a second. I understand that its flashy and y'all want to make a statement.

[WCAG 2.3.1](https://www.w3.org/TR/WCAG20/#seizure)
[Understanding WCAG 2.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html)
[How to Meet WCAG 2.3.1](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0#qr-seizure-does-not-violate)

**Comment by code3z** [comment]
Yeah, gun.eco isn't really accessible, whether you have disabilities or not. The mouseover effect should really just be a small flicker. And it is basically unusable on mobile.

These issues are [especially bad](https://www.reddit.com/r/webdev/comments/qpc1iy/comment/hjt4z9q/?utm_source=share&utm_medium=web2x&context=3) considering Gun is trying to appeal to webdevs, especially frontend devs.
